extension/smsdk_config.h

```diff
@@ -40,7 +40,7 @@
 #define SMEXT_CONF_NAME			"Accelerator"
 #define SMEXT_CONF_DESCRIPTION	"SRCDS Crash Handler"
 #define SMEXT_CONF_VERSION		SM_FULL_VERSION
-#define SMEXT_CONF_AUTHOR		"Asher \"asherkin\" Baker"
+#define SMEXT_CONF_AUTHOR		"Asher \"asherkin\" Baker" " | " "Benoist \"Kenzzer\" André"
```

Comment:
Might as well fix this while we're here.

```suggestion
#define SMEXT_CONF_AUTHOR		"Erin \"asherkin\" Baker" ", " "Benoist \"Kenzzer\" André"
```

Although I think I'd probably drop it down to just:
```suggestion
#define SMEXT_CONF_AUTHOR		"asherkin, Kenzzer"
```